cmd/lakectl/cmd/branch.go

```diff
@@ -116,7 +116,7 @@ var branchRevertCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```

Comment:
```suggestion
		fmt.Println("Branch: %s\n", u)
```

Reply:
kept the format and dropped the call to String()